src/chain_spec.rs

```diff
@@ -115,6 +123,23 @@ fn polymath_props() -> Properties {
         .clone()
 }
 
+fn currency_codes() -> Vec<Ticker> {
+    // Fiat Currency Struct
+    #[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+    #[derive(Encode, Decode, Clone, Debug, PartialEq, Eq)]
+    pub struct FiatCurrency<String> {
+        pub codes: Vec<String>,
+    }
+
+    let currency_file = fs::read_to_string("src/data/currency_symbols.json").unwrap();
```

Comment:
@JMoore96 This line is triggering the panic here: https://vi4cae9kel5o.polyme.sh/teams/main/pipelines/polymesh-develop-pr/jobs/build-polymesh/builds/683#L5f5d187f:318

This happens because this file read is happening at runtime, which will be relative to somewhere in `target/` (wherever the `polymesh` binary ends up), as opposed to at compile time. To hard-code the file at compile-time, you can use https://doc.rust-lang.org/nightly/std/macro.include_str.html.

Reply:
alright, would start working on the change